va/va.c

```diff
@@ -439,6 +442,7 @@ static VAStatus va_openDriver(VADisplay dpy, char *driver_name)
                     CHECK_VTABLE(vaStatus, ctx, BeginPicture);
                     CHECK_VTABLE(vaStatus, ctx, RenderPicture);
                     CHECK_VTABLE(vaStatus, ctx, EndPicture);
+                    CHECK_VTABLE(vaStatus, ctx, MFSubmit);
                     CHECK_VTABLE(vaStatus, ctx, SyncSurface);
```

Comment:
The new functions are not mandatory to implement, could you remove the checks for the new functions ?

Reply:
Done